create-database.sql

```diff
@@ -1,52 +1,49 @@
 CREATE DATABASE ShopDB;
+
 USE ShopDB;
 
+
 CREATE TABLE Countries (
-    ID INT,
-    Name VARCHAR(50),
+    ID INT AUTO_INCREMENT,
+    Name VARCHAR(50) NOT NULL,
     PRIMARY KEY (ID)
 );
 
 CREATE TABLE Products (
     ID INT AUTO_INCREMENT,
-    Name VARCHAR(50),
+    Name VARCHAR(50) NOT NULL,
     PRIMARY KEY (ID)
 );
 
 CREATE TABLE Warehouses (
     ID INT AUTO_INCREMENT,
-    Name VARCHAR(50),
-    Address VARCHAR(50),
-    CountryID INT,
+    Name VARCHAR(50) NOT NULL,
+    Address VARCHAR(50) NOT NULL,
+    CountryID INT NOT NULL,
 	FOREIGN KEY (CountryID) REFERENCES Countries(ID) ON DELETE NO ACTION,
     PRIMARY KEY (ID)
 );
 
 CREATE TABLE ProductInventory (
-    ID INT,
-    ProductID INT,
+    ID INT AUTO_INCREMENT,
+    ProductID INT NOT NULL,
     FOREIGN KEY (ProductID) REFERENCES Products(ID) ON DELETE NO ACTION,
-    WarehouseAmount INT,
-    WarehouseID INT,
+    WarehouseAmount INT NOT NULL,
+    WarehouseID INT NOT NULL,
     FOREIGN KEY (WarehouseID) REFERENCES Warehouses(ID) ON DELETE NO ACTION,
     PRIMARY KEY (ID)
 );
 
--- Populate test data
 INSERT INTO Countries (ID,Name)
 	VALUES (1, 'Country1');
```

Comment:
When inserting into a table with an AUTO_INCREMENT field, it's generally unnecessary to specify the ID value. The database will automatically generate this value. Consider removing the ID from the insert statement.